assets/fastload/lang/zh_tw.json

```
@@ -0,0 +1,28 @@
+{
+  "fastload.screen.buildingTerrain.template": "已完成／目標",
+  "fastload.screen.buildingTerrain.building": "正在建構可見區塊",
+  "fastload.screen.buildingTerrain.preparing": "正準備所有區塊",
+
+  "fastload.screen.config": "快速載入設定",
+
+  "fastload.button.debug": "除錯模式",
+  "fastload.button.debug.tooltip": "要將除錯模式的訊息在記錄檔中顯示嗎？",
+
+  "fastload.button.close_loading_screen_unsafely": "關閉載入畫面",
+  "fastload.button.close_loading_screen_unsafely.tooltip": "快速載入模組應該提前關掉「正在載入地形」的畫面，即使它稍微有點不安全？",
```

```suggestion
  "fastload.button.close_loading_screen_unsafely.tooltip": "Fastload 是否應該提前關掉「正在載入地形」的畫面，即使會有點不安全？",
```
使用模組的英文名稱應該會更好一點，另外 `稍微` 跟 `有點` 有點多餘

---

感謝建議！
如果這個模組有要 PR 的話，也會將你加入為協作者
（我還在思考這個可行性，包括審核者與協作者應與一起作為協作者 Co Author PR 回原始模組專案）

---

@xMikux 我記得 Github 會自動處理這部分變成協作者